utilities.py

```diff
@@ -224,25 +224,56 @@ def save_roi_signals(
 
         dataset_name = defs.DoricFile.Dataset.ROI.format(str(id_).zfill(4))
 
-        if roi_ids is None:
+        attrs = {
+            defs.DoricFile.Attribute.ROI.ID:           id_,
+            defs.DoricFile.Attribute.ROI.SHAPE:        0,
+            defs.DoricFile.Attribute.ROI.COORDS:       coords,
+            defs.DoricFile.Attribute.Dataset.NAME:     defs.DoricFile.Dataset.ROI.format(id_),
+            defs.DoricFile.Attribute.Dataset.USERNAME: defs.DoricFile.Dataset.ROI.format(id_)
+        }
+
+        if attrs_add is not None:
+            attrs = {**attrs, **attrs_add}
+
+        save_signal(signals[i], f, f"{path}/{dataset_name}", attrs)
+
+    save_signal(time_, f, f"{path}/{defs.DoricFile.Dataset.TIME}")
+
+
+def save_spike_signals(
```

Comment:
Hmm.. spikes are not necessarily named as ROI 1, ROI 2, etc., so not sure if it's the best way to organize things..
But spikes are always named the same way as the input signals, so I would create the names only once and pass it to the `save_roi_signals()` and `save_signals()` (to save spikes).